src/frontend/src/common/components/library/data_subview/components/EditSamplesConfirmationModal/index.tsx

```diff
@@ -1,7 +1,9 @@
 import CloseIcon from "@material-ui/icons/Close";
-import React, { useMemo } from "react";
+import _ from "lodash";
```

Comment:
nit: why not `import { pick } from lodash;`?

Reply:
fixed